net-libs/tox/tox-9999.ebuild

```diff
@@ -19,23 +19,24 @@ RESTRICT="!test? ( test )"
 REQUIRED_USE="^^ ( log-debug log-error log-info log-trace log-warn )
 		daemon? ( dht-node )"
 
-COMMON_DEPEND="
-	av? ( media-libs/libvpx
-		media-libs/opus )
-	daemon? ( acct-group/tox
-		acct-user/tox
-		dev-libs/libconfig )
-	>=dev-libs/libsodium-0.6.1:=[asm,urandom,-minimal]"
 BDEPEND="virtual/pkgconfig"
-
-DEPEND="${COMMON_DEPEND}"
-
-RDEPEND="${COMMON_DEPEND}"
+DEPEND=">dev-libs/libsodium-0.6.1:=[asm,urandom,-minimal]
+	av? (
+		media-libs/libvpx:=
+		media-libs/opus
+	)
+	daemon? ( dev-libs/libconfig )"
+RDEPEND="
+	${DEPEND}
+	daemon? (
+		acct-group/tox
+		acct-user/tox
+	)"
 
 src_prepare() {
 	cmake_src_prepare
 	#remove faulty tests
-	for testname in bootstrap lan_discovery save_compatibility tcp_relay tox_many_tcp; do
+	for testname in lan_discovery save_compatibility; do
```

Comment:
Above this, add a line: ```local testname=```.

Reply:
I am not entirely certain what you mean. I will include one possible interpretation in my updated pull request.